### PR TITLE
update rancher-operator to v0.1.3-rc4

### PR DESCRIPTION
**Pull request by kinarashah**
https://github.com/rancher/rancher/issues/29993

### CODE REVIEW

**Comment by StrongMonkey** [comment]
We shouldn't need to generate patch file..

File: packages/rancher-operator-crd/package.yaml [hunk 1]
```diff
@@ -1,3 +1,3 @@
-url: https://github.com/rancher/rancher-operator/releases/download/v0.1.3-rc3/rancher-operator-crd-0.1.3-rc3.tgz
+url: https://github.com/rancher/rancher-operator/releases/download/v0.1.3-rc4/rancher-operator-crd-0.1.3-rc4.tgz
 packageVersion: 00
-releaseCandidateVersion: 03
```
Comment (StrongMonkey) [inline packages/rancher-operator-crd/package.yaml:3]: We should remove this field `releaseCandidateVersion`

Comment (kinarashah) [reply packages/rancher-operator-crd/package.yaml:3]: remove the field or not update it? I'm using https://github.com/rancher/charts/pull/967/files as an example

**Review by StrongMonkey — APPROVED** [review]

File: packages/rancher-operator-crd/package.yaml (hunk 1 above)
Comment (StrongMonkey) [reply packages/rancher-operator-crd/package.yaml:3]: nvm

**Pull request merged**